src/test/protobuf/oldRepeatables.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```

Comment:
can this be moved to `src/test/protobuf/custom_collection.proto`, and have the collection_type applied to only the field, not the whole file?